### PR TITLE
Add `Environment.namespaceToName()` method

### PR DESCRIPTION
**Pull request by arthurvr**
Fix https://github.com/yeoman/environment/issues/20

### CODE REVIEW

**Comment by SBoudrias** [comment]
Should this be a static method or an instance method?

**Comment by supergray** [comment]
PRs stay open for 17 days :-1: 

> Should this be a static method or an instance method?

not tied to a specific environment instance so static. go read about OOP

**Comment by arthurvr** [comment]
> PRs stay open for 17 days :-1:

W00t? I opened it yesterday.

**Comment by SBoudrias** [comment]
Sorry for the troll...

Anyway, we should double check where the call site for a similar function would be and see if it's more convenient to have it as an instance method rather than a static one.

**Comment by arthurvr** [comment]
> Anyway, we should double check where the call site for a similar function would be and see if it's more convenient to have it as an instance method rather than a static one.

I guess it makes more sense as an instance method. I added a static one for now as that seemed what the original issue's title suggests, but am fine with it being an instance method. What would be the most handy in `yo`'s case?

**Comment by SBoudrias** [comment]
Looks okay to keep this as a static method.

File: lib/environment.js [hunk 1]
```diff
@@ -452,6 +452,17 @@ Environment.createEnv = function (args, opts, adapter) {
 };
 
 /**
+ * Convert a generators namespace to its name
+ *
+ * @param  {Namespace} namespace
```
Comment (SBoudrias) [inline lib/environment.js]: Namespace is not a valid type, we want a string here.

Comment (arthurvr) [inline lib/environment.js]: Haha whoops! Updated.

**Comment by SBoudrias** [comment]
Awesome, thanks!

**Pull request merged**